Automattic/jetpack
# Tools: Add tampermonkey script for Jetpack Live Branches on GitHub

Introduces a script to be used at will with the Tampermonkey (or similar) browser extension


#### Changes proposed in this Pull Request:

* Adds a directory `tools/jetpack-live-branches` with a README and the`jetpack-live-branches.js` script.


#### Testing instructions:


* Click https://github.com/Automattic/jetpack/raw/add/jetpack-live-branches-tampermonkey-script/tools/jetpack-live-branches/jetpack-live-branches.user.js.
* Visit a PR on this repo
* Expect to see a block of text added to the PR description allowing you to launch a site with a given Pull Request already set up on the site.

#### Screenshot

![image](https://user-images.githubusercontent.com/746152/38027630-bfd88320-3266-11e8-8485-27677e2dcf15.png)



#### Proposed changelog entry for your changes:
